test/terraform_aws_ecs_example_test.go

```diff
@@ -0,0 +1,66 @@
+package test
+
+import (
+	"fmt"
+	"terratest/modules/aws"
+	"terratest/modules/random"
+	"terratest/modules/terraform"
```

Comment:
The test failed due to an import error here. I missed this in the first go around, but when using `dep`, the source is checked out under `github.com/gruntwork-io/terratest`, which means these need to include that prefix. E.g `terratest/modules/aws` => `github.com/gruntwork-io/terratest/modules/aws`.

Reply:
I've fixed the imports, but when I run `dep ensure` it adds terratest to the lock file. Is this behavior expected? 
See my reverted commit: https://github.com/gruntwork-io/terratest/pull/283/commits/8369f7e3520e4bcadf962fad242dddcf69d69133

Reply:
Hmm that is unexpected. It does not happen for me when I run `dep ensure`. What version of `dep` are you using? Run `dep version` to see.

Reply:
@yorinasub17 nevermind, it works fine if the project is cloned into 
`${GOPATH}/src/github.com/gruntwork-io/terratest`

golang is not my primary language, sorry :)

Reply:
Ah yes that is a common gotcha in Go: the significance of `GOPATH`.

Reply:
yup, I thought it was unexpected too, that's why I've reverted the commit. 
Any plans on migrating to `go mod`?

Reply:
You can see our recent discussion on `go mod` for our current thought process: https://github.com/gruntwork-io/terratest/pull/264